javascript/javascript1/week1/startupname.js

```diff
@@ -0,0 +1,5 @@
+let firstWords = ["Samsung", "Apple", "Sony", "Moto", "OnePlus", "Microsoft", "RedHat", "Ford", "Mitsubishi", "Tesla"];
+let lastWords = ["Galaxy", "IPhone", "Experia", "Razor", "Nord", "Windows", "Linux", "Fiesta", "SpaceStar", "Model S"];
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
const randomNumber = Math.floor(Math.random() * firstWords.length) which gives random value between 0 and length of an array